python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -943,7 +943,7 @@ def testSubdividePolygonsAttribute( self ) :
 		for subdividePolygons in ( None, False, True ) :
 			a = IECore.CompoundObject()
 			if subdividePolygons is not None :
-				a["ai:polymesh:subdividePolygons"] = IECore.BoolData( subdividePolygons )
+				a["ai:polymesh:subdivide_polygons"] = IECore.BoolData( subdividePolygons )
```

Comment:
There was method in the madness of the existing naming - "subdividePolygons" isn't a native Arnold attribute but one we synthesise ourselves, so it was following the Cortex convention as a small reminder of that fact. I think we discussed this way back when, but maybe you feel this is definitely better?

Reply:
Well, the thing is that diffuse_reflect isn't a native Arnold name either - we use it to set a bitmask flag named AI_RAY_DIFFUSE_REFLECT, so to me both subdividePolygons/diffuseReflect or subdivide_polygons/diffuse_reflect are equally names we're synthesizing ourselves, and I feel like they ought to be consistent?

Reply:
True, although DIFFUSE_REFLECT does at least map directly to an Arnold concept of the same name, whereas subdivide_polygons isn't even really an Arnold concept. Still, I don't feel like I'm on very strong ground here, so if you think this is definitely better that's fine with me...